src/ngraph/frontend/onnx_import/op/elu.cpp

```diff
@@ -46,18 +38,8 @@ namespace ngraph
                     std::shared_ptr<ngraph::Node> alpha_node =
                         std::make_shared<ngraph::op::Constant>(
                             data->get_element_type(), Shape{}, std::vector<double>{alpha});
-                    alpha_node = ngraph::op::make_broadcast_node(alpha_node, data->get_shape());
 
-                    std::shared_ptr<ngraph::Node> zero_node =
-                        std::make_shared<ngraph::op::Constant>(
-                            data->get_element_type(), Shape{}, std::vector<double>{0});
-                    zero_node = ngraph::op::make_broadcast_node(zero_node, data->get_shape());
-
-                    return {std::make_shared<ngraph::op::Maximum>(data, zero_node) +
-                            alpha_node *
-                                std::make_shared<ngraph::op::Exp>(
-                                    std::make_shared<ngraph::op::Minimum>(data, zero_node)) -
-                            alpha_node};
+                    return {std::make_shared<ngraph::op::Elu>(data, alpha_node)};
```

Comment:
Please remove the `{` and `}'

Reply:
These are needed to return the `NodeVector` with a single node.

Reply:
I think it'd be clearer in this case to do it as `return NodeVector{std::make_shared<ngraph::op::Elu>(data, alpha_node)};`

Reply:
Done.